### PR TITLE
Return 1 if publishing fails

### PR DESCRIPTION
**Pull request by rshaiju**
The tool is modified to return 1 if publishing has any error. On success, it will return 0

### CODE REVIEW

File: src/Kmd.Logic.Gateway.Automation.Tool/PublishCommandHandler.cs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ﻿using System;
```
Comment (Araballi) [inline src/Kmd.Logic.Gateway.Automation.Tool/PublishCommandHandler.cs:1]: Fix the warning

Comment (rshaiju) [reply src/Kmd.Logic.Gateway.Automation.Tool/PublishCommandHandler.cs:1]: fixed

**Review by Araballi — COMMENTED** [review]
Remove warning which is not required, line no 10 and 12

**Review by Araballi — APPROVED** [review]
LGTM

**Pull request merged**